src/registrar/models/domain.py

```diff
@@ -1062,6 +1062,15 @@ def is_expired(self):
         now = timezone.now().date()
         return self.expiration_date < now
 
```

Comment:
[posting as a code comment so we can thread convo]
@rachidatecs In general everything looks great! Just a few more small edits for the counter: 
- label should use lower case, so "23 domains" and "6 domain requests"
- for the counter font, use "base-dark" (that's a request from Gaby!)
- raise the counter so it aligns with the top of the pagination buttons
- only show the counter if there's 1 or more domains/requests

Reply:
Done